storybook/src/admin-date-time/DatePicker.tsx

```diff
@@ -12,7 +12,7 @@ const Story = () => {
     }
 
     const initialValues: Partial<Values> = {
-        dateOne: null,
+        dateOne: new Date(),
```

Comment:
Why is this necessary?

Reply:
The clearable button is only visible when there is a Date set. That's why I changed this. I can add a third Date in to show the empty state. Should I do that?